### ISSUE TITLE
Can antd enhance the actual performance of components in the project by adding e2e testing?

### ISSUE THREAD

**Issue by kenjiding** [ant-design/ant-design#49481]
### What problem does this feature solve?

I found that other component libraries use e2e regression testing to seriously affect the actual performance of components. Can Antd also achieve this

### What does the proposed API look like?

Antd can also implement e2e regression testing

**Comment by leekilee** [comment]
I agree, component libraries mostly rely on unit tests for testing, and components are hard to integration testing. However, E2E testing can address this issue and enhance the effectiveness of components in real-world projects.
